new-frontiers-14/frontier-station-14
# Fill states for the Frontier chemistry and medicine wall lockers.

## About the PR



Added fill states for the medicine and chemistry wall lockers made by Erhard.

If it's not considered out of line I would also like it to be considered allright to remove the block on the hand labeler. They're currently mapped on all ships that use chemistry, but it is a piece of chemistry equipment that's expected to be used. Odd that it's removed.

## Why / Balance


Upstream has an empty and a filled variant. Seemed fitting that we have a empty and filled variant as well. Erhard said go nuts.

## How to test


Place lockers. Check fills. Make sure they're anchored and the parents are behaving.

## Media


![ss+(2024-10-01+at+07 18 42)](https://github.com/user-attachments/assets/843bb8b0-8a31-40fb-a267-0b45a62cecc1)

## Requirements



## Breaking changes


None.

**Changelog**


🆑 
- add: Added filled medicine and chemistry Frontier wall lockers.
